### PR TITLE
add back es head

### PR DESCRIPTION
**Pull request by snopoke**
@millerdev

### CODE REVIEW

**Comment by millerdev** [comment]
What's the purpose of this? Would it make sense to put a comment in *docker/Dockerfile_es* explaining why we need it?

**Comment by millerdev** [comment]
Also probably makes sense to change [docker/.dockerignore](https://github.com/dimagi/commcare-hq/blob/b5d369773eb652789e2c5160791d8fd809e6c3e8/docker/.dockerignore):
```diff
- /Dockerfile_commcarehq_base
+ /Dockerfile_*
```

**Comment by snopoke** [comment]
It's  gives a web UI that's useful for accessing ES locally and our prod
clusters.

**Comment by millerdev** [comment]
Cool. What port is that exposed on? Could be useful to document that in Dockerfile_es to make it a bit more discoverable.

**Comment by czue** [comment]
it's a plugin that runs with elasticsearch. you should be able to access it locally at http://localhost:9200/_plugin/head/

File: docker/Dockerfile_es [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM elasticsearch:1.7.4
+
+# Install HEAD plugin
+RUN plugin -i mobz/elasticsearch-head
```
Comment (esoergel) [inline docker/Dockerfile_es]: :+1: Just noticed this missing.

**Comment by esoergel** [comment]
Tested it out locally.  Looks good to me!

File: docker/hq-compose-services.yml [hunk 2]
```diff
@@ -20,14 +20,16 @@ redis:
...
 elasticsearch:
-  extends:
-    file: hq-compose.yml
-    service: elasticsearch
+  build: .
+  dockerfile: Dockerfile_es
   command: elasticsearch --cluster.name=${ES_CLUSTER_NAME}
   environment:
-    - ES_CLUSTER_NAME
+    ES_CLUSTER_NAME: ES_CLUSTER_NAME
```
Comment (millerdev) [inline docker/hq-compose-services.yml]: Should this say `ES_CLUSTER_NAME:: "${ES_CLUSTER_NAME}"`

**Pull request merged**